django_ewiz/decompiler.py

```diff
@@ -136,11 +142,18 @@ def __request_multiple(self, url, count_only=False):
                 id_list.append(pattern.match(smart_str(line)).group('value'))
 
             count = int(id_list[0])
-            idList = id_list[1:]
+            id_list = id_list[1:]
```

Comment:
Please change the first ``id_list`` to ``response_list`` to match the first condition branch.